### PR TITLE
New bundle: Pathology MIL classification from WSI 

### LINKED ISSUE #5587: Add a new bundle to model zoo: pathology MIL bundle
Add a new bundle to model zoo for Multiple Instance Learning (MIL) classification from Whole Slide Images (WSI), similar to the tutorial in https://github.com/Project-MONAI/tutorials/tree/main/pathology/multiple_instance_learning

### PR DESCRIPTION
**Pull request by myron**
Fixes https://github.com/Project-MONAI/MONAI/issues/5587

This adds  a baseline model for Multiple Instance Learning (MIL) classification from Whole Slide Images (WSI), similar to the tutorial https://github.com/Project-MONAI/tutorials/tree/main/pathology/multiple_instance_learning

### CODE REVIEW

**Comment by Nic-Ma** [comment]
Hi @yiheng-wang-nv ,

Please note that this new version should not be updated to NGC, thanks.

File: models/endoscopic_inbody_classification/configs/metadata.json [hunk 1]
```diff
@@ -1,8 +1,7 @@
 {
     "schema": "https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/meta_schema_20220324.json",
-    "version": "0.3.4",
+    "version": "0.3.3",
```
Comment (Nic-Ma) [inline models/endoscopic_inbody_classification/configs/metadata.json]: I don't understand why change the version of this endoscopy back?
@wyli Do you know any context? I feel it's not our development proposal.

Thanks.

Comment (yiheng-wang-nv) [reply models/endoscopic_inbody_classification/configs/metadata.json]: I think it comes from the old version of the dev branch, there are same questions on ci files. Hi @myron , may need to sync with the latest dev branch, and solve the conflicts, thanks!

**Comment by yiheng-wang-nv** [comment]
HI @myron , for the `models/pathology_multiple_instance_learning_classification/scripts/datalist_panda_0.json`, is it possible to create a script that can produce the datalist (like https://github.com/Project-MONAI/model-zoo/blob/dev/models/brats_mri_segmentation/scripts/prepare_datalist.py does), or upload the file into https://github.com/Project-MONAI/MONAI-extra-test-data/releases/tag/0.8.1 (like https://github.com/Project-MONAI/model-zoo/tree/dev/models/lung_nodule_ct_detection/docs#10-fold-data-splitting)

**Comment by myron** [comment]
> HI @myron , for the `models/pathology_multiple_instance_learning_classification/scripts/datalist_panda_0.json`, is it possible to create a script that can produce the datalist (like https://github.com/Project-MONAI/model-zoo/blob/dev/models/brats_mri_segmentation/scripts/prepare_datalist.py does), or upload the file into https://github.com/Project-MONAI/MONAI-extra-test-data/releases/tag/0.8.1 (like https://github.com/Project-MONAI/model-zoo/tree/dev/models/lung_nodule_ct_detection/docs#10-fold-data-splitting)

it's possible to upload it to another place, e.g. https://github.com/Project-MONAI/MONAI-extra-test-data/releases/tag/0.8.1
But I don't understand why we do it for datalist files, it's just a list of filenames.  is it wrong to just include this small file inside of the bundle together with other custom scripts?

File: models/endoscopic_inbody_classification/configs/metadata.json (hunk 1 above)
Comment (myron) [reply models/endoscopic_inbody_classification/configs/metadata.json]: hmm, this must be a mistake.  this PR is only about MIL bundle, and shouldn't touch anything else. let me take a look

Comment (myron) [reply models/endoscopic_inbody_classification/configs/metadata.json]: okay, I've removed those changes (I'm not sure why/how it happened)

**Comment by yiheng-wang-nv** [comment]
> > HI @myron , for the `models/pathology_multiple_instance_learning_classification/scripts/datalist_panda_0.json`, is it possible to create a script that can produce the datalist (like https://github.com/Project-MONAI/model-zoo/blob/dev/models/brats_mri_segmentation/scripts/prepare_datalist.py does), or upload the file into https://github.com/Project-MONAI/MONAI-extra-test-data/releases/tag/0.8.1 (like https://github.com/Project-MONAI/model-zoo/tree/dev/models/lung_nodule_ct_detection/docs#10-fold-data-splitting)
> 
> it's possible to upload it to another place, e.g. https://github.com/Project-MONAI/MONAI-extra-test-data/releases/tag/0.8.1 But I don't understand why we do it for datalist files, it's just a list of filenames. is it wrong to just include this small file inside of the bundle together with other custom scripts?

Hi @myron , the main purpose is for maintain. If saving to a common place, we only need to modify it once (rather than modify both tutorial and bundle). 

I just checked the tutorial side, and the link of google drive is attached rather than adding the datalist file (https://github.com/Project-MONAI/tutorials/tree/main/pathology/multiple_instance_learning#train). Therefore, I suggest to use the same way.

**Comment by myron** [comment]
> > > HI @myron , for the `models/pathology_multiple_instance_learning_classification/scripts/datalist_panda_0.json`, is it possible to create a script that can produce the datalist (like https://github.com/Project-MONAI/model-zoo/blob/dev/models/brats_mri_segmentation/scripts/prepare_datalist.py does), or upload the file into https://github.com/Project-MONAI/MONAI-extra-test-data/releases/tag/0.8.1 (like https://github.com/Project-MONAI/model-zoo/tree/dev/models/lung_nodule_ct_detection/docs#10-fold-data-splitting)
> > 
> > 
> > it's possible to upload it to another place, e.g. https://github.com/Project-MONAI/MONAI-extra-test-data/releases/tag/0.8.1 But I don't understand why we do it for datalist files, it's just a list of filenames. is it wrong to just include this small file inside of the bundle together with other custom scripts?
> 
> Hi @myron , the main purpose is for maintain. If saving to a common place, we only need to modify it once (rather than modify both tutorial and bundle).
> 
> I just checked the tutorial side, and the link of google drive is attached rather than adding the datalist file (https://github.com/Project-MONAI/tutorials/tree/main/pathology/multiple_instance_learning#train). Therefore, I suggest to use the same way.

ok, I've set it up to download json file from the g-drive, @yiheng-wang-nv plz check

**Comment by myron** [comment]
@yiheng-wang-nv , question about bundles.  What is the process for a user to download files in large_files.yml ? (such as pretrained weights) is there a command e.g. `python -m monail.bundle download large files` ??

**Comment by yiheng-wang-nv** [comment]
> @yiheng-wang-nv , question about bundles. What is the process for a user to download files in large_files.yml ? (such as pretrained weights) is there a command e.g. `python -m monail.bundle download large files` ??

Hi @myron , this format is only used by developers, and after a bundle is merged, all large files will be downloaded into the defined path and a whole bundle will be compressed for users to download directly.

**Comment by yiheng-wang-nv** [comment]
Hi @myron , your bundle will be in compatible with new bundle interface (WIP, designed by Nic in the ticket mentioned above). Do you mind close this PR now, and reopen (may also need to update) it later? Thanks!

**Pull request closed without merging**